src/main/java/org/kitteh/irc/client/library/defaults/DefaultBuilder.java

```diff
@@ -343,7 +370,9 @@ public Client build() {
 
         Client.WithManagement client = new DefaultClient();
         client.initialize(this.name, this.getInetSocketAddress(this.serverHost, this.serverPort), this.serverPassword,
-                this.getInetSocketAddress(this.bindHost, this.bindPort), this.nick, this.userString, this.realName,
+                this.getInetSocketAddress(this.bindHost, this.bindPort),
+                this.getInetSocketAddress(this.proxyHost, this.proxyPort), this.proxyType,
```

Comment:
You're checking if the proxy functionality is enabled by checking if `address` is not `null`, while `getInetSocketAddress` will *never* return a non-`null` value.